migrations/versions/922ee620c114_.py

```diff
@@ -0,0 +1,45 @@
+"""empty message
+
+Revision ID: 922ee620c114
+Revises: 37436458bc78
+Create Date: 2020-04-30 13:13:22.284555
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "922ee620c114"
+down_revision = "37436458bc78"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
```

Comment:
not sure why this is here.

Reply:
This is not intentional - I too am unsure. I'll delete the file.